toxcore/network.c

```diff
@@ -1357,11 +1410,23 @@ int net_connect(Socket sock, IP_Port ip_port)
         return 0;
     }
 
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+    return 0;
+#else
     return connect(sock.socket, (struct sockaddr *)&addr, addrsize);
+#endif
 }
 
 int32_t net_getipport(const char *node, IP_Port **res, int tox_type)
 {
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+    *res = (IP_Port *)malloc(sizeof(IP_Port));
```

Comment:
Might want to assert this allocation succeeds. Assertion failures are nicer than segfaults.